### PR TITLE
add hosted services docs

### PR DESCRIPTION
**Pull request by devoncarew**
@lukechurch, @sethladd, can you review? Some docs on setting up hosted systems. The `coverage.md` files was inadvertently committed as part of another PR; please review as well.

### CODE REVIEW

File: doc/webdriver.md [hunk 1]
```diff
@@ -1,5 +1,95 @@
...
-- TODO: saucelabs
+## Integration testing
+
+So let's say you have a web app you're working on. You might have a continuous
+build set up and running on a service like Travis. Let's say you have some unit
+tests written and they're running on each commit on your build system. If you're
+very forward looking you might be using a service like coveralls.io to get a
+sense of the code coverage from those unit tests.
+
+So you've got a continuous build, and tests, and maybe code coverage. But what
```
Comment (sethladd) [inline doc/webdriver.md]: "So you have got"... can just be "Imagine you have a continuous..."

**Comment by sethladd** [comment]
lgtm

File: doc/webdriver.md (hunk 1 above)
Comment (devoncarew) [inline doc/webdriver.md]: done -

**Comment by lukechurch** [comment]
lgtm

**Pull request merged**